lib/task_bunny/config.ex

```diff
@@ -88,6 +88,18 @@ defmodule TaskBunny.Config do
     end)
   end
 
+  # Checks worker configuration sanity.
+  @spec worker_enabled?(keyword) :: boolean
+  defp worker_enabled?(queue) do
+    case Keyword.get(queue, :worker, []) do
+      false -> false
```

Comment:
need `nil -> @default_concurrency` too.

Reply:
I mean `nil -> true`

Reply:
That's implicit. Since it is either set or `[]` (the default on line `94`)
Then doing `Keyword.get(worker, :concurrency, @default_currency)` (with `worker` = `[]` does end up with the default concurrency.

Reply:
ah, cool. i was missing line 94.